src/webview/columns/address-column.tsx

```diff
@@ -31,7 +31,7 @@ export class AddressColumn implements ColumnContribution {
     render(range: BigIntMemoryRange, _: Memory, options: TableRenderOptions): ReactNode {
         return <span className='memory-start-address'>
             {options.showRadixPrefix && <span className='radix-prefix'>{getRadixMarker(options.addressRadix)}</span>}
-            <span className='address'>{getAddressString(range.startAddress, options.addressRadix, options.effectiveAddressLength)}</span>
+            <span className='address hoverable' data-column='address'>{getAddressString(range.startAddress, options.addressRadix, options.effectiveAddressLength)}</span>
```

Comment:
Sorry I missed this before, but could we make the whole span hoverable? I noticed that when I hover over the radix prefix (e.g., `0x`) then the hover does not show up which feels a bit odd.

Reply:
OK.